subliminal/extensions.py

```diff
@@ -100,7 +102,7 @@ def unregister(self, entry_point):
 ])
 
 #: Disabled providers
-disabled_providers = ['addic7ed', 'napiprojekt', 'opensubtitlesvip', 'shooter']
+disabled_providers = ['addic7ed', 'napiprojekt', 'opensubtitles', 'opensubtitlesvip', 'shooter']
```

Comment:
can we wait a little bit with this one? ( #1068 )

Reply:
kept the old API provider